src/components/navigation/pagination/NavigationPaginationReducers.ts

```diff
@@ -37,6 +38,10 @@ export const paginationReducer = (state: IPaginationState = paginationInitialSta
             if (contains(state.id, action.payload.id) && action.payload.shouldResetPage) {
                 return {...state, pageNb: 0};
             }
+        case FilterActions.filterThrough:
+            if (contains(state.id, action.payload.id)) {

```

Comment:
i'd suggest using `s.contains` instead of direct imports but noticed it's not you who did that, probably me actually 😄 ,  not a mandatory change but you can do so if you feel like it